buildchain/buildchain/iso.py

```diff
@@ -188,21 +189,36 @@ def action(targets: Sequence[str]) -> None:
 @doit.create_after(executed='populate_iso')  # type: ignore
 def task__iso_build() -> types.TaskDict:
     """Create the ISO from the files in ISO_ROOT."""
-    mkisofs = [
-        config.ExtCommand.MKISOFS.value, '-output',  ISO_FILE,
-        '-quiet',
-        '-rock',
-        '-joliet',
-        '-joliet-long',
-        '-full-iso9660-filenames',
-        '-volid', '{} {}'.format(config.PROJECT_NAME, versions.VERSION),
-        '--iso-level', '3',
-        '-gid', '0',
-        '-uid', '0',
-        '-input-charset', 'utf-8',
-        '-output-charset', 'utf-8',
-        constants.ISO_ROOT
-    ]
+    def unlink_if_exist(filepath: Path) -> None:
```

Comment:
IIRC we have a couple of these now (I recall @ycointe inserted something similar). Could be a `utils` candidate.

Reply:
Yup, but the one you mention is out of the buildchain so not that easy to share.